Sources/XcodeGenKit/PBXProjGenerator.swift

```diff
@@ -798,16 +806,24 @@ public class PBXProjGenerator {
             }
             
             // set Carthage search paths
+            let configFrameworkBuildPaths: [String]
             if !carthageDependencies.isEmpty {
-                let frameworkSearchPaths = "FRAMEWORK_SEARCH_PATHS"
                 let carthagePlatformBuildPath = "$(PROJECT_DIR)/" + getCarthageBuildPath(platform: target.platform)
+                configFrameworkBuildPaths = [carthagePlatformBuildPath] + Array(frameworkBuildPaths).sorted()
```

Comment:
I thought it was best that Carthage path was first then the rest alphabetically. If it should be strictly alphabetic let me know.

Reply:
All good 👍